qa/suites/rados/upgrade/jewel-x-singleton/o

```diff
@@ -0,0 +1,6 @@
+[HANDLER_OUTPUT] 
```

Comment:
@liewegas teuthology-suite panics at seeing this file, as it is not ended with  `.yaml` and this directory will be empty without this file. which renders the test matrix representing this directory an empty one:
```
Traceback (most recent call last):
  File "/home/kchai/teuthology/virtualenv/bin/teuthology-suite", line 11, in <module>
    load_entry_point('teuthology', 'console_scripts', 'teuthology-suite')()
  File "/home/kchai/teuthology/scripts/suite.py", line 137, in main
    return teuthology.suite.main(args)
  File "/home/kchai/teuthology/teuthology/suite/__init__.py", line 88, in main
    run.prepare_and_schedule()
  File "/home/kchai/teuthology/teuthology/suite/run.py", line 309, in prepare_and_schedule
    num_jobs = self.schedule_suite()
  File "/home/kchai/teuthology/teuthology/suite/run.py", line 476, in schedule_suite
    build_matrix(suite_path, subset=self.args.subset)
  File "/home/kchai/teuthology/teuthology/suite/build_matrix.py", line 50, in build_matrix
    mat, first, matlimit = _get_matrix(path, subset)
  File "/home/kchai/teuthology/teuthology/suite/build_matrix.py", line 60, in _get_matrix
    mat = _build_matrix(path, mincyclicity=outof)
  File "/home/kchai/teuthology/teuthology/suite/build_matrix.py", line 122, in _build_matrix
    fn)
  File "/home/kchai/teuthology/teuthology/suite/build_matrix.py", line 122, in _build_matrix
    fn)
  File "/home/kchai/teuthology/teuthology/suite/build_matrix.py", line 131, in _build_matrix
    return matrix.Sum(item, submats)
  File "/home/kchai/teuthology/teuthology/suite/matrix.py", line 225, in __init__
    "Sum requires non-empty _submats"
AssertionError: Sum requires non-empty _submats
```

Reply:
fixed!